app/src/main/java/com/example/concordia_campus_guide/Activities/PathsActivity.java

```diff
@@ -121,7 +137,8 @@ public void showInfoCard() {
     }
 
     /**
-     *  checks if from and to are indoor places and stores in a boolean array {from, to}
+     * checks if from and to are indoor places and stores in a boolean array {from, to}
+     *
      * @param from
      * @param to
```

Comment:
Despite the clear parameter namings, you should write a small explanation for each since u have a  javac doc for the mehtod